tensorly/decomposition/tests/test_parafac2.py

```diff
@@ -472,3 +473,47 @@ def test_parafac2_to_tensor():
         Bi = T.dot(projections[i], factors[1])
         manual_tensor = T.einsum("r,jr,kr", factors[0][i], Bi, factors[2])
         assert_(tl.max(tl.abs(constructed_tensor[i, :, :] - manual_tensor)) < 1e-6)
+
+
+def test_update_imputed():
```

Comment:
The unit test is nice, but it would also be good with an integration test where we mask out some values and check that we get the correct components by running PARAFAC2 ALS EM

Reply:
Added this test on 174eca66f9bcb0daca88df7862e48e805817214e. Thanks for the suggestion! :)